workflow-ui/core-common/src/main/java/com/squareup/workflow1/ui/ViewEnvironment.kt

```diff
@@ -41,6 +49,9 @@ constructor(
 
   override fun hashCode(): Int = map.hashCode()
 
+  @Suppress("UNCHECKED_CAST")
+  private fun <T : Any> getOrNull(key: ViewEnvironmentKey<T>): T? = map[key] as? T
```

Comment:
Worth to make it `getOrDefault`, as all callers are using some sort of fallback.

Reply:
But the main call wants to run a lambda only in the non-null case.